### DISCUSSION TITLE
Is there any way to run gravity-sync?

### DISCUSSION THREAD

**Discussion by cristacheda** [DesktopECHO/Pi-Hole-for-WSL1#101] [Q&A] [answered]
I used your script to install Pi Hole for WSL and I would really like to sync this Pi Hole instance with another one that I'm running on a Raspberry Pi. How should I go about using [gravity-sync](https://github.com/vmstan/gravity-sync)?

**Comment by DesktopECHO** [comment, marked as answer]
If you're running a debian-based distro (like Raspbian) you can use `C:\Program Files\Pi-hole\Gravity Sync.cmd`

You'll be prompted with a question:

`Replication partner's first sync will 'pull' records from this Pi-hole, or 'push' records to this Pi-hole? [pull]:`

If you want your "real" Pi-hole's configuration sent to a fresh install of Pi-hole for Windows, answer **push** 

It will create a script which you copy/paste into a remote SSH session on the gravity-sync partner.  The script downloads gravity-sync and performs pre-requisite setup:

![image](https://github.com/DesktopECHO/Pi-Hole-for-WSL1/assets/33142753/f74defad-1d4f-4e4f-b7e9-1ed2000d05e5)

Note that sync jobs are configured and run on the partner side, and are never run from Windows.  
From your Raspberry Pi, run `sudo gravity-sync info` to verify things look ok:

![image](https://github.com/DesktopECHO/Pi-Hole-for-WSL1/assets/33142753/67d97261-b89d-4fb7-8a36-5ed1b2cde102)

I haven't heard much feedback on this funcionality so let me know if it's working for you.

**Reply by cristacheda** [reply]
Everything seems to be working. I will update in a few days if anything goes wrong. Thanks a lot!
